server/src/main/java/org/elasticsearch/common/time/DateFormatters.java

```diff
@@ -466,7 +466,7 @@
 
     private static final DateTimeFormatter STRICT_BASIC_WEEK_DATE_PRINTER = new DateTimeFormatterBuilder()
         .parseStrict()
-        .appendValue(IsoFields.WEEK_BASED_YEAR, 4)
+        .appendValue(IsoFields.WEEK_BASED_YEAR, 4, 10, SignStyle.NEVER)
```

Comment:
why SignStyle.NEVER?
I guess we could try with NORMAL (this is default)

Reply:
Sure, I think I took this from elsewhere but NORMAL sounds right to me.